usaspending_api/api_contracts/contracts/autocomplete/programSourceComponents/AgencyIdentifier.md

```diff
@@ -0,0 +1,42 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Agency Identifier (AID)
+
+This endpoint powers USAspending.gov's Treasury Account and Federal Account Agency Identifier (AID) component filter in the Advanced Search Program Source (TAS) filter.
+
+## Agency Identifier [/api/v2/autocomplete/accounts/aid/]
+
+Returns lists of possible AIDs matching the search string and narrowed down by the given component filters.
+
+## POST
++ Request
+    + Attributes (object)
+        + filters (required, ComponentFilters)
```

Comment:
Do we want to support a `limit` or should we just always return everything?  (The endpoint currently has a `limit` following the city search autocomplete model,  but obviously it can be removed.)

Reply:
We should keep the limit